SofascoreAcademy/SofascoreAcademy/LeagueView.swift

```diff
@@ -84,3 +71,42 @@ extension UIStackView {
         }
     }
 }
+
+extension LeagueView: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return leaguesData.count
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return leaguesData[section].matches.count + 1
```

Comment:
```suggestion
       leaguesData[section].matches.count + 1
```

Kad je oneliner moze bez return-a